### ISSUE TITLE
factorial, '!' is not pronounced

### ISSUE THREAD

**Issue by LongBoolean** [MycroftAI/mycroft-core#150]
"Mycroft what is five factorial?"
Mycroft: "five is one hundred and twenty"
The '!' is not pronounced, likely because it is treated like a regular exclamation mark by the TTS. 
Either mimic needs to learn how to differentiate between the meanings, something that could be a bit error prone when applied generically, or the wolfram skill should convert "5!" into "5 factorial" before calling speak.

```
2016-06-15 15:34:01,022 - SpeechClient - INFO - Utterance: [u'is 5 factorial']
2016-06-15 15:34:01,035 - SpeechClient - INFO - Speak: I am searching for is 5 factorial
2016-06-15 15:34:01,043 - METRICS - DEBUG - {"attributes": {"version": "dev-78b917a", "utterances": ["is 5 factorial"]}, "levels": {}, "timers": {"mycroft.stt.remote.time_s": [2.6535191535949707, 3.8827672004699707], "mycroft.cerberus.proxy.client.time_s": [2.6535191535949707, 3.8827672004699707], "mycroft.recognizer.extractor.time_s": [0.15184402465820312], "mycroft.recognizer.audio.length_s": [4.928], "mycroft.recognizer.audio_extracted.length_s": [1.717, 2.718], "mycroft.stt.local.time_s": [0.06133890151977539, 0.025575876235961914, 0.008630037307739258, 0.01871514320373535, 0.019158124923706055, 0.020533084869384766, 0.019608020782470703, 3.814697265625e-05, 0.010708093643188477, 0.005048036575317383, 0.008708000183105469, 0.006637096405029297, 0.007827043533325195]}, "counters": {}}
2016-06-15 15:34:04,281 - SpeechClient - INFO - Speak: 5! is 120
2016-06-15 15:34:12,217 - SpeechClient - INFO - Listening...
```

**Comment by forslund** [comment]
I would recommend that the wolfram skill converts the !-operator to a string. As you say, in Mimic it'll be really hard to define the case where it should be expanded to "factorial" and when it should be dropped.

**Comment by ethanaward** [comment]
This is fixed by #152 and can be closed.
